### PR TITLE
#210 add case annotation display functionality

### PR DESCRIPTION
**Pull request by andrewtavis**
Hey @SaurabhJamadagni! 😊

Firstly, I didn't do the conference, so don't think that I can get to all this while doing something like that at the same time 😄

What this is:
- Disables the annotations for nouns as they don't do anything (can't be clicked and don't have shadows)
- Switches the keyboard to the conjugation display when a preposition annotation is pressed
- Allows for migration through the cases via the left and right buttons in the display
- Makes the first case shown to the users the first one in the annotation
    - All cases will be able to be seen from the display with the left and right keys though
- Adds variables for German and Russian case conjugation displays
- Renaming variables :)

### CODE REVIEW

**Comment by andrewtavis** [comment]
[efccc13](https://github.com/scribe-org/Scribe-iOS/pull/215/commits/efccc131aef8275f8a9ddf9db460dbaf77473a6e) is just to add the autosuggest files for the other issue so that I can have them available on the current branch for testing as I try to generate them :)

**Comment by andrewtavis** [comment]
@SaurabhJamadagni, [40c094f](https://github.com/scribe-org/Scribe-iOS/pull/215/commits/40c094f6dbfb3883997538985d39f44de0af9fe2) is likely gonna be the dumbest PR you ever get from me, and might be because it's late, but easter eggs are fun, so why not? 🐣😊 Let me know what you think 😄

Along with that are some formatting fixes to the ranges as you suggested, as well as adding `"Scribe"` to the nouns so that it shows up in auto actions :)

File: Keyboards/LanguageKeyboards/French/Data/autosuggestions.json [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (SaurabhJamadagni) [inline Keyboards/LanguageKeyboards/French/Data/autosuggestions.json:1]: Just nit picking but extra unnecessary whitespace 😅

**Review by SaurabhJamadagni — APPROVED** [review]
Hey @andrewtavis! This is such a cool feature! Definitely doesn't feel like a WIP implementation. Also, loved the Scribe easter egg! Looks good to me and didn't experience any bugs during the limited testing I did. Thanks for the review! This was a fun feature to check out. Has me excited 😊

File: Keyboards/LanguageKeyboards/French/Data/autosuggestions.json (hunk 1 above)
Comment (andrewtavis) [reply Keyboards/LanguageKeyboards/French/Data/autosuggestions.json:1]: I'm honestly really excited too 😊 This is gonna be so great :) :)

Thanks for checking it all!
